CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - [#725](https://github.com/plotly/dash-core-components/pull/725) Improve async graph performance by parallelizing resource fetching instead of fetching sequentially
 
 ### Updated
+- [#732](https://github.com/plotly/dash-core-components/pull/732)
+  - Upgraded plotly.js to [1.52.1](https://github.com/plotly/plotly.js/releases/tag/v1.52.1)
+  - [Feature release 1.52.0](https://github.com/plotly/plotly.js/releases/tag/v1.52.0) which contains:
+    - Enable loading locale bundles before plotly.js bundles [#4453](https://github.com/plotly/plotly.js/pull/4453)
+    - `ko` localization [#4315](https://github.com/plotly/plotly.js/pull/4315)
```

Comment:
@alexcjohnson Added these based on previous entries (ko) and because of its significance for async/Dash. Feel free to add other items that are relevant.

Reply:
That's fine, good call pointing out the ordering fix. Aside from that, there are no new trace types in 1.52, that's the most important thing to include.